crates/torii/core/src/engine.rs

```diff
@@ -172,17 +177,19 @@ impl<P: Provider + Send + Sync + std::fmt::Debug> Engine<P> {
     pub async fn fetch_data(
         &mut self,
         from: u64,
-        pending_block_tx: Option<Felt>,
+        last_pending_block_world_tx: Option<Felt>,
```

Comment:
I wonder if there would be a way to do this with only one tx hash, perhaps we try to index the tx hashes and then only sync range from > tx hash

Reply:
hmm not sure i understand